### DISCUSSION TITLE
Port 443 and 80 already in use

### DISCUSSION THREAD

**Discussion by Shiori9** [nextcloud/all-in-one#5081] [Questions] [answered]
I'm running a minecraft Pterodactyl panel on the same server i want to install Nextcloud on. Mind i have 0 knowledge on all of this works but essentially the panel uses port 80 and 443 with Nextcloud needing to use the same unable to change. I want to know if or how to jsut get it to work. The best i did was running the start up script without a reverse proxy in place. I could access the AIO through port 8081 but it started complaining about not having a reverse prozy in palce. The last thing i did was https://github.com/ichiTechs/Dockerized-SSL-NextCloud-with-MariaDB.git , which isn't doing nothing. It just reroutes my NC domain to my Pterodactyl panel.

**Comment by Zoey2936** [comment, marked as answer]
you can run aio and pterodactyl behind NPMplus, but I will only try to help with NPMplus related problems, so please understand that I won't tell you how to install aio/pterodactyl behind a reverse proxy

**Reply by Shiori9** [reply]
I have set up NPMPlus aswell as nextcloud but i cant get ssl certificats to work. i set them up and proxy them right but they jsut dont encrypt

**Reply by Zoey2936** [reply]
Logs? Error message?
